### PR TITLE
[FIX] web: list: clarify "select all" feature

### PR DESCRIPTION
**Pull request by aab-odoo**
This PR contains several fixes involving the "select all" feature in list view.

Task~4150362

### CODE REVIEW

File: addons/web/i18n/web.pot [hunk 1]
```diff
@@ -4815,6 +4815,13 @@ msgstr ""
 #: code:addons/web/static/src/legacy/xml/base.xml:0
 #: code:addons/web/static/src/views/view_dialogs/export_data_dialog.xml:0
```
Comment (Polymorphe57) [inline addons/web/i18n/web.pot:4816]: several several pages

Comment (Polymorphe57) [reply addons/web/i18n/web.pot:4816]: we simply this
commit

**Comment by Polymorphe57** [comment]
@robodoo r+

**Review by aab-odoo — COMMENTED** [review]
robodoo rebase-ff

**Pull request closed without merging**